Harvest.Web/ClientApp/src/Expenses/BilledExpensesListContainer.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+import { CommonRouteParams, Expense, Project } from "../types";
+import { ExpenseTable } from "./ExpenseTable";
+
+import { authenticatedFetch } from "../Util/Api";
+
+import { useIsMounted } from "../Shared/UseIsMounted";
+import { ProjectHeader } from "../Shared/ProjectHeader"; //TODO add this once it is tested and running
+
+interface Props {
+  newExpenseCount?: number; // just used to force a refresh of data when new expenses are created outside of this component
+}
+
```

Comment:
used anywhere?

Reply:
Probably not, this was copy pasta, and I wasn't sure what it was doing.